my_list.rb

```diff
@@ -0,0 +1,32 @@
+require './my_enumerable'
```

Comment:
Great job @itsmraga-hub 🤝

- [ ] Kindly use `require_relative` to import `MyEnumerable` since using `require_relative` for files in your directory instead of `require` is a ruby convention. "Require should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory."